pkg/controller/validatingadmissionpolicystatus/OWNERS

```diff
@@ -1,6 +1,9 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+approvers:
+  - sig-api-machinery-approvers
 reviewers:
   - jiahuif
```

Comment:
probably also emeritus?

Reply:
SG.  I think emeritus is approver only, so I'll just remove.